build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1231,6 +1249,18 @@ message PriorityCapabilities {
   repeated PriorityRange priorities = 1;
 }
 
+// Describes how the server treats absolute symlink targets.
+enum SymlinkAbsolutePathStrategy {
+  ABSOLUTE_SYMLINK_UNKNOWN = 0;
```

Comment:
Grrr, C++ and its stupid scoping rules! How about we move the enum under CacheCapabilities message instead? Our current enums ignore the C++ use case in their naming, and I'd like to be consistent.

Reply:
Fine by me. Update pushed.